src/common.rs

```diff
@@ -279,6 +288,24 @@ impl From<Rva> for u32 {
     }
 }
 
+impl Add<u32> for Rva {
+    type Output = Self;
+
+    /// Adds the given offset to this `Rva`.
+    fn add(mut self, offset: u32) -> Self {
+        self.0 += offset;
+        self
+    }
+}
+
+impl Sub for Rva {
+    type Output = u32;
+
+    fn sub(self, other: Self) -> Self::Output {
+        self.0 - other.0
```

Comment:
Should `Rva`/`PdbInternalRva` provide `checked_sub()` instead?  Is the underflow condition worth detecting?

This is part of the public API, both directly and indirectly via `AddressMap`. Someone is going to subtract two `Rva`s in the wrong order, which seems like a bug, which seems like a thing we could detect and signal here. Is underflow always an error? Is it always the library user's error? What, if anything, should we do about it?

Reply:
I've added implementations for `checked_`, `saturating_` and `wrapping_` now, certainly not bad to have them (I was just lazy initially). I would treat this just like the primitive `u32`: If someone wants to perform these checks the methods are there, but for cases where this is not necessary they can be skipped.

As to whether over-/undeflows are errors, I think this is up to the context. Just yesterday we ran into an overflow when adding an RVA offset but then figured out that the value we were reading was bogus (one of those marker values again...). So I would leave this up to the user.